packages/contracts/src/RelayHub.sol

```diff
@@ -192,11 +192,29 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
     function withdraw(uint256 amount, address payable dest) public override {
         address payable account = payable(msg.sender);
         require(balances[account] >= amount, "insufficient funds");
+        uint256[] memory amounts = new uint256[](1);
+        address payable[] memory destinations = new address payable[](1);
+        amounts[0] = amount;
+        destinations[0] = dest;
+        withdrawMultiple(amounts, destinations);
+    }
 
-        balances[account] = balances[account].sub(amount);
-        dest.transfer(amount);
-
-        emit Withdrawn(account, dest, amount);
+    /// @inheritdoc IRelayHub
+    function withdrawMultiple(uint256[] memory amount, address payable[] memory dest) public override {
+        address payable account = payable(msg.sender);
+        for (uint256 i = 0; i < amount.length; i++) {
+            // #if ENABLE_CONSOLE_LOG
```

Comment:
I still think that adding these logs does't help a bit, only add to clutter the logs. sending out 4 log lines adds even more. can use `log( "string with %s", a,b,c)` (though it is limited only to 3 dynamic params)

Reply:
and while you're at it, add "ifdef" around the "import console.log" too.

Reply:
Fixed.